src/test/java/com/example/document/DocumentEntryApplicationTests.java

```diff
@@ -9,7 +9,7 @@
 @SpringBootTest
 @ExtendWith(SpringExtension.class)
 @ActiveProfiles("test")
-class DocumentApplicationTests {
+class DocumentEntryApplicationTests {
```

Comment:
I think this test just checks that application is started
And it should be named just `DocumentApplicationTests` as before